packages/server/lib/controllers/connection.controller.ts

```diff
@@ -34,6 +34,11 @@ class ConnectionController {
             const instantRefresh = req.query['force_refresh'] === 'true';
             const isSync = (req.get('Nango-Is-Sync') as string) === 'true';
 
+            if (!providerConfigKey) {
```

Comment:
should we also error if it is an empty string?

Reply:
it's already taken care of thanks to javascript